### ISSUE TITLE
sklearn-(min_groups, self.n_splits)), Warning)

### ISSUE THREAD

**Issue by XufengXufengXufeng** [Microsoft/LightGBM#1177]
Please search your question on previous issues, [stackoverflow](https://stackoverflow.com/questions/tagged/lightgbm) or other search engines before you open a new one.

For bugs and unexpected issues, please provide following information, so that we could reproduce on our system.

## Environment info
Operating System: Windows 10
CPU: intel i7
C++/Python/R version: python 3.6
Pandas: 0.20.1
Numpy: 1.12.1
Sklearn: 0.18.1

## Error Message:
![image](https://user-images.githubusercontent.com/31204817/34594493-657d1d9e-f19f-11e7-9024-ace0c9887cbb.png)
I keep getting this error. I am not sure how it happens to me but not to the other users, as I am using it in such a normal way.
## Reproducible examples

I loaded data like this. The train_data is a pandas dataframe, that I got from a csv file.

dtrain = lgb.Dataset(train_data.drop('Sales',axis=1).values,train_data.Sales.values)
![image](https://user-images.githubusercontent.com/31204817/34594685-92cb7be6-f1a0-11e7-96aa-6187620b8d3e.png)

then I did a lgb.cv like this:

param = {
    'objective':'quantile',
    'max_depth':8,
    'boosting_type':'gbdt',
    'feature_fraction':0.9,
    'bagging_fraction':0.8,
    'num_leaves':31,
    'verbosity':0,
    'learning_rate':0.05,
    'bagging_freq':5
}
hist = lgb.cv(
    param,
    dtrain,
    early_stopping_rounds = 30,
    metrics = 'l1',
    nfold = 3,
    verbose_eval = False,
    num_boost_round = 20
    )

and then it happened, every time.
## Steps to reproduce

1. load data from a pandas dataframe but with .values that returns a numpy.array type.
2. then set everything for a lgb.cv test.
3. then run the test and see if the error appears

I am pretty sure that I have done something wrong somehow, and it is not a system error. It's just that there are so little online materials about the usage of lgb comparing with the xgb and it is indeed faster than the xgb. Please help.

**Comment by StrikerRUS** [comment]
@XufengXufengXufeng You are using `cv` for regression problem (your objective is `quantile`), so you need to change the default value of `stratified : bool, optional (default=True)` param - set it to `False`.

**Comment by XufengXufengXufeng** [comment]
thank you that was the issue. I should look into the parameters more carefully next time.

**Comment by StrikerRUS** [comment]
You are welcome!
